docs/wallets/desktop_wallet.md

```diff
@@ -0,0 +1,217 @@
+
+- **Step 1** - Select the "*Send*" on the left panel, fill up the request information: 
+    - recipient address
+    - amount to be sent 
+    - optional memo
+        <img src="./assets/desktop_wallet/2-1-1.png" />
+
+- **Step 2** - Once you hit the "*Continue*" button, you can review the transaction details in the confirmation window on the Desktop Wallet and proceed to the next step by clicking "*Confirm*". Then enter your app password and hit "*Decrypt Wallet*".
+        <img src="./assets/desktop_wallet/2-1-2.png" />
+
+- **Step 3** - In the "*Confirm Transaction*" page, please double-check the information and make sure the information you entered is correct. Now you can go with "*Confirm*" and click "*OK*" on the next page.
+        <img src="./assets/desktop_wallet/2-1-3.png" />
+
+- **Step 4** - Now your funds have been transferred successfully to your selected address! You can then check the transaction history under “Transaction” at the “Home” page.
+        <img src="./assets/desktop_wallet/2-1-4.png" />
+        <img src="./assets/desktop_wallet/2-1-5.png" />
+
+### 2.2 - Receive Transactions
+If someone is sending funds or you are sending funds from your other account to the current account, you can simply show the bar code or the address of your current account.
+        <img src="./assets/desktop_wallet/2-2.png" />
+
+## 3. Staking Operations
+Crypto.org Chain is based on Tendermint Core's consensus engine, it relies on a set of validators to participate in the proof of stake (PoS) consensus protocol who are responsible for committing new blocks in the blockchain. Token holders might stake their tokens to validators to secure the network and earn rewards. Particularly, we can:
+- Delegate to a validator;
+- Withdraw staking rewards from a validator;
+- Undelegate from a validator.
+
+### 3.1 - Delegate to a validator
+- **Step 1** - To begin staking, please direct to "*Delegate Funds*" under the Staking option. You can choose one of the active validators that you would like to delegate to: 
+        <img src="./assets/desktop_wallet/3-1-1-1.png" />
+You can input the validator’s information either by:
+    - Clicking the list button under "*Validator address*", and selecting your preferred validators. 
+        <img src="./assets/desktop_wallet/3-1-1-2.png" />
+
+    - Copying the complete address of your preferred validator, and pasting the validator’s "*Operator Address*" (begin with `tcrocncl...` for testnet) under "*Validator address*". Then specify the amount that you would like to delegate.
+
+**Remark**: You can look up validators and their operator address by the validator list on the explorer.
```

Comment:
Can link to [maininet validator explorer](https://crypto.org/explorer/validators)/[testnet validator explorer](https://crypto.org/explorer/croeseid3/validators)